src/About/About.test.tsx

```diff
@@ -1,21 +1,23 @@
-import React from "react";
-import { render, screen, getByAltText} from "@testing-library/react";
-import About from "./About";
-import { MemoryRouter } from "react-router-dom";
+import React from 'react'
+import { render, screen } from '@testing-library/react'
```

Comment:
I promise i'm not putting double quotes in there on purpose!!!!!

Reply:
Oh I know! It happens to me all the time with VSCode. I just grab em when I'm in that file!